apps/google-analytics-4/lambda/package.json

```diff
@@ -11,13 +11,14 @@
     "start:dev": "nodemon",
     "test": "TS_NODE_TRANSPILE_ONLY=1 mocha",
     "test:debug": "TS_NODE_TRANSPILE_ONLY=1 mocha -- --inspect --inspect-brk",
-    "test:watch": "TS_NODE_TRANSPILE_ONLY=1 mocha --watch --watch-files src --watch-files test"
+    "test:watch": "TS_NODE_TRANSPILE_ONLY=1 mocha --watch --watch-files src --watch-files test",
+    "deploy:test": "sls deploy --stage test",
+    "deploy": "sls deploy --stage $STAGE"
```

Comment:
The most important line to review. Please double check this makes sense to you too!